icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2039,6 +2052,61 @@ class U_I18N_API NumberFormatterSettings {
      */
     Derived scale(const Scale &scale) &&;
 
+#ifndef U_HIDE_DRAFT_API
+    /**
+     * Specifies the usage for which numbers will be formatted ("person",
+     * "road", "person", etc.)
+     *
+     * When a `usage` is specified, the output unit will change depending on the
```

Comment:
Question: is there a way to iterate through (or get a hash set of) the usages? If not, that makes it tricky to know what ones are available : and to ensure that the right terms are used (eg 'person' instead of 'people' or 'human') and spelled the same.

Reply:
No; that's a limitation of the proposed API. It is a "best effort".

Reply:
I've made a note somewhere that I want to link to some listing of usages in these docs. The best option I know of (having not researched this, and not created something better) is to link to the xml or the units.txt file on github.